etl/src/main/java/org/pih/hivmigration/etl/sql/LabResultMigrator.groovy

```diff
@@ -7,7 +7,6 @@ class LabResultMigrator extends ObsMigrator {
         executeMysql("Create staging table", '''
             create table hivmigration_lab_results (
               obs_id int PRIMARY KEY AUTO_INCREMENT,
-              source_patient_id int,
```

Comment:
I did this for some migrators, but not all of them. Hence leaving `source_encounter_id` in the tmp_obs table for backwards-compatibility.